.github/workflows/image-scan-analysis.yml

```diff
@@ -1,61 +1,705 @@
+      OPENSHIFT_NAMESPACE: 3cd915-tools
+      output-filename: nodejs14.txt
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Authenticate and set context
+      uses: redhat-actions/oc-login@v1
+      env:
+      # These can be stored in secrets, if desired.
+        OPENSHIFT_USER: ${{secrets.OC_USERNAME_SA}} # service account gitaction in openshift will be used here!
+        OPENSHIFT_NAMESPACE: 3cd915-tools
+        SERVER_URL: ${{secrets.OPENSHIFT_SERVER}}
+        OPENSHIFT_REGISTRY: ${{secrets.OPENSHIFT_REGISTRY}} 
+
+      with:
+      # URL to your OpenShift cluster.
+      # Refer to Step 2.
+        openshift_server_url: ${{ secrets.OPENSHIFT_SERVER }}
 
+      # Authentication Token. Can use username and password instead.
+      # Refer to Step 3.
+        openshift_token: ${{ secrets.OPENSHIFT_TOKEN }}
+    
+    - name: Login to OpenShift Container Registry
+      uses: docker/login-action@v1
+      with:
+        registry: ${{secrets.OPENSHIFT_REGISTRY}}
+        username: ${{ secrets.OC_USERNAME_SA }}
+        password: ${{ secrets.OPENSHIFT_TOKEN }}
+    
+    - name: Pull nodejs Base Image
+      env:
+        OPENSHIFT_NAMESPACE: 3cd915-tools
+      run: |
+        #docker login -u ${{env.OPENSHIFT_USER}} -p $(oc whoami -t) ${{env.OPENSHIFT_REGISTRY}}   
+        docker pull ${{secrets.OPENSHIFT_REGISTRY}}/${{env.OPENSHIFT_NAMESPACE}}/${{env.image-name}}:${{env.tag}} 
+        #docker image ls
+      working-directory: ${{env.working-directory}}
+    - name: Scan nodejs14 Base Image with Aqua Trivy
+      id: scan
+      uses: aquasecurity/trivy-action@master
+      with:
+        image-ref: '${{secrets.OPENSHIFT_REGISTRY}}/${{env.OPENSHIFT_NAMESPACE}}/${{env.image-name}}:${{env.tag}}'
+        format: 'table'
+        exit-code: '1'
+        ignore-unfixed: true
+        vuln-type: 'os,library'
+        severity: 'CRITICAL,HIGH'
+        output: ${{env.output-filename}}
+    - if: failure() && steps.scan.outcome == 'failure'
+      name: Upload artifact
+      uses: actions/upload-artifact@v2
+      with:
+        name: ${{env.image-name}}_scan_report
+        path: ./${{env.output-filename}}
+        retention-days: 3
+        if-no-files-found: error
+  
+  comment_nodejs_pr:
+  # this action will leave a comment in response to vulnerability scans performed on cotnainer image
+    if: always() && needs.nodejs-base-image.result == 'failure' && github.event_name == 'pull_request'
+    needs: nodejs-base-image
+    runs-on: ubuntu-latest
+    env:
+      working-directory: ./frontend
+      image-name: nodejs-14-ubi8
+      tag: ${{secrets.NODEJS14_BASE_TAG}}
+      OPENSHIFT_NAMESPACE: 3cd915-tools
+      output-filename: nodejs14.txt
+    steps:
+      - uses: actions/checkout@v2
+      - name: Download artifact
+        id: artifact
+        uses: actions/download-artifact@master
+        with:
+         name: ${{env.image-name}}_scan_report
+         path: ./${{env.image-name}}
```

Comment:
if the purpose of this step is to download the file created by the trivy scan, should this not be: 
./${{env.output-filename}} to match line 79?